src/loader.js

```diff
@@ -216,6 +216,38 @@
   };
 
   // 4. Registry Objects
+  
+  // 4.2.1
+  // For now, registry objects are a work in progress that don't fully integrate into the rest of the code base
+  function Registry(loader) {
+    if (!this.constructor)
+      throw new TypeError('The Registry constructor must be called with "new"');
```

Comment:
I put it in because it's part of the spec (4.2.1 #1), but am happy to take it out if you'd prefer.